src/main/kotlin/com/example/demo/infrastructure/mapper/TodoMapper.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.demo.infrastructure.mapper
+
+import org.apache.ibatis.annotations.Mapper
+import com.example.demo.model.TodoModel
+import org.springframework.stereotype.Component
+import java.util.*
+import java.util.concurrent.atomic.AtomicReference
+
+@Mapper
+@Component
+interface TodoMapper {
+
+    //todoDB
+    fun find(): List<TodoModel>
+    fun getTodoById(id:String): TodoModel
+    fun insert(id:String, title:String, categoryId:Int, detail:String, statusId:Int, deadline: Date, remarks:String): String
```

Comment:
バラバラに受け取らない。
TodoModelの型で受け取る
```suggestion
    fun insert(todoModel: TodoModel): String
```